tests/time_tests/test_runner/conftest.py

```diff
@@ -235,6 +236,9 @@ def pytest_runtest_makereport(item, call):
     data["error_msg"] = ""
 
     data = {field: data[field] for field in FIELDS_FOR_SUBMIT}
+    assert MODEL_FIELDS.issubset(data["model"].keys()), \
```

Comment:
Should enforce this through JSON schema?

Reply:
Good idea, I reimplemented it with `jsonschema`. Please, check new version